oci/config/convert/runtime.go

```diff
@@ -148,11 +147,11 @@ func MutateRuntimeSpec(g rgen.Generator, rootfs string, image ispec.Image) error
 	}
 
 	// Set optional fields
-	var ports []string
-	for port := range image.Config.ExposedPorts {
-		ports = append(ports, port)
+	ug, err := ugen.NewFromImage(image)
```

Comment:
I agree, it just makes sense. Then we could also change the interface of this function and pass an image generator instead of the image. Any objections?